src/ansys/fluent/core/utils/fluent_version.py

```diff
@@ -115,11 +117,29 @@ def get_latest_installed(cls):
             If an Ansys version cannot be found.
         """
         for member in cls:
-            if member.awp_var in os.environ:
+            if member.awp_var in os.environ and member.get_fluent_exe_path().exists():
                 return member
 
         raise AnsysVersionNotFound(
-            "Verify the value of the 'AWP_ROOT' environment variable."
+            "Unable to locate a compatible Ansys Fluent installation. "
```

Comment:
@mkundu1 Noting that we have three places where theis exception type is raised. One is version-specific and two are not. We can consolidate the message creation in the class itself.